documentation/deliverables/individual_deliverables/open-source-group-kickoff.md

```diff
@@ -0,0 +1,148 @@
+# Open Source Group Kickoff
+
+| Field           | Value                                                        |
+| --------------- | ------------------------------------------------------------ |
+| Document Status | Draft                                                        |
+| Epic Link       | [Issue 72](https://github.com/HHS/grants-equity/issues/72)   |
+| Epic Dashboard  | [Milestone Roadmap](https://github.com/orgs/HHS/projects/12) |
```

Comment:
Can you update this to match the [summary table in the other deliverable specs](https://github.com/HHS/simpler-grants-gov/blob/main/documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md#static-site-public-launch-infrastructure-and-content)?

Reply:
done! Billy, I wasn't sure if I should mark you as a product owner or as another document owner since you would be contributing to this document too. Feel free to change it.

Reply:
This isn't as relevant if we'll be moving these into the public wiki!